_fixtures/testvariables3.go

```diff
@@ -0,0 +1,20 @@
+package main
```

Comment:
> I am glad you made the fix to hide this and make such test case unnecessary.

Note that it could still happen, either because of corrupted memory or because of non-live in-scope variables (although that would be pretty unlikely).

Reply:
Yes of course. The code still has extra checks.